src/ec/suite_b/ops/p256.rs

```diff
@@ -88,14 +88,14 @@ fn p256_elem_inv(a: &ElemUnreduced) -> ElemUnreduced {
     }
 
     let b_1 = &a;
-    let b_11     = sqr_mul(&b_1,  0 +  1, &b_1);
+    let b_11     = sqr_mul(b_1,   0 +  1, b_1);
```

Comment:
I did this one intentionally, but OK, if Clippby complains, let's get rid of it.